src/igvfd/schemas/changelogs/human_donor.md

```diff
@@ -1,10 +1,12 @@
-## Changelog for human_donor.json
+## Changelog for *`human_donor.json`*
 
-### Minor changes since version 1
+### Minor changes since schema version 1
 
-* *taxon_id* now restricted to NCBI taxonomy ids that start with NCBI:txid followed by numbers. Example NCBI:txid9606 for *Homo sapiens*
-* *taxon_id* replaced by organism
-* defined *sex* as required property and defaulted it to *unspecified*
-* human_donor has aliases mixin and submitted mixin removed, but still has properties for aliases and submitted due to change of basic_item mixin.  The actual properties have not changed.
-* organism property renamed to taxa
-* Human donors now have a property *traits* to specify known phenotypic traits
+* Add `traits`.
+* Rename `organism` to `taxa`.
```

Comment:
Will we add the newest changes at the top or bottom of the lists? Just wondering if it is confusing here (since we happened to rename the same property twice). If we are consistent about adding to the top of the list, then this seems OK.

Reply:
yes I have tried consistently to put latest changes on the top